### PR TITLE
Draft notification MVP entry added

### PR DESCRIPTION
**Pull request by jesseyuen**
Let me know what you think @fofr! 🦅 👀

### CODE REVIEW

File: app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+title: Notifications MVP
+description: Identifying an initial set of notifications and the critical user flows required to manage notifications in Publish.  
+date: 2020-01-23
+---
+UCAS provided email notifications to it's users. This functionality is yet be introduced to Publish teacher training courses. To address this, we established an introductory set of notifications to satisfy our users critical needs.
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md]:
```suggestion
UCAS provided email notifications to it's users. This functionality is yet be introduced to Publish teacher training courses. To address this, we established an introductory set of notifications to satisfy our user’s critical needs.
```

File: app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+## User needs
+
+**As an accredited body**
+I need to be notified of changes by providers in Publish to courses I accredit
+So that I can keep my student record system up to date and in sync with UCAS.
+
+## Critical notifications
+
+Events that would initiate a notification were identified.
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md]:
```suggestion
Events that would initiate a notification were identified:
```

File: app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+1. A course has a basic detail change
+2. A course is created
+3. A course is published
+4. A course is withdrawn
+5. There are no more vacancies in a course (different from withdrawing)
+6. A courses accredited body has changed
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md]:
```suggestion
6. A course’s accredited body has changed
```

File: app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+The first two templates were to be included in the MVP.
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md]: Not sure what this means

File: app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+## Critical user journeys
+
+User journeys required to facilitate notifications were identified.
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md]:
```suggestion
User journeys required to facilitate notifications were identified:
```

File: app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+1. The recipient needs to be able to opt in and opt out of notifications.
+2. The recipient needs to be able to update notifications settings (change email address etc).
+
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md]:
```suggestion
The recipient needs to be able to:

* opt in and opt out of notifications
* update notifications settings (change email address etc)

```

File: app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+{% endset %}
+
+{% from "email/macro.njk" import appEmail %}
+{{ appEmail({
+  subject: "Course [Course name] (code) has been updated",
+  content: template1
+}) }}
+
+## Template - A course has been created
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md]:
```suggestion
## Email template: A course has been created
```

File: app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+## Critical user journeys
+
+User journeys required to facilitate notifications were identified.
+
+1. The recipient needs to be able to opt in and opt out of notifications.
+2. The recipient needs to be able to update notifications settings (change email address etc).
+
+These tasks were deemed outside of our MVP and would be managed manually, with users contacting support, [becomingateacher@digital.education.gov.uk](mailto:becomingateacher@digital.education.gov.uk).
+
+## Template - A course has a basic detail change
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md]:
```suggestion
## Email template: A course has a basic detail change
```

**Review by fofr — COMMENTED** [review]
A couple of grammar/wording suggestions.

Looks good – only one bit I wasn't sure about.

File: app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md (hunk 4 above)
Comment (jesseyuen) [reply app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md]: Good shout.  

> The first two templates were to be included in the MVP. 

...only makes sense in context of the pilot which is the next entry for Notifications.

**Pull request merged**